shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -490,6 +490,7 @@ public void onPostResume() {
   protected void onNewIntent(@NonNull Intent intent) {
     // Forward Intents to our FlutterFragment in case it cares.
     flutterFragment.onNewIntent(intent);
+    super.onNewIntent(intent);
```

Comment:
I don't think this requirement is correct. The docs don't say that the super call through is required, and the implementation here is a no-op:
https://android.googlesource.com/platform/frameworks/base/+/android-2.3.5_r1/core/java/android/app/Activity.java#1025

I'd like to avoid calling super in cases where it doesn't do anything.